test/e2e/monitoring_stack_controller_test.go

```diff
@@ -110,9 +113,9 @@ func TestMonitoringStackController(t *testing.T) {
 	}, {
 		name:     "Alertmanager deployed and removed",
 		scenario: assertAlertmanagerDeployedAndRemoved,
```

Comment:
Either add a brief explanation for this, or rebase. From what we discussed IIUC you were waiting for https://github.com/rhobs/observability-operator/pull/312 to get merged, weren't you?